src/shared/components/in-app-search/InAppSearch.tsx

```diff
@@ -57,6 +57,7 @@ export type Props = {
   genomeId: string;
   genomeIdForUrl: string; // this should be a temporary measure; it should be returned by search api
   mode: InAppSearchMode;
+  size?: InputSize;
```

Comment:
I don't understand your approach to InAppSearch:

- If it truly exists in only two sizes, then why is the `size` field optional?
- However, does it really need the `size` prop? Isn't the `mode` prop sufficient? Aren't all the interstitial instances of InAppSearch large and all sidebar instances small?

Reply:
I did lean towards using `mode` initially but then thought we might see the pattern breaking in future designs and went with the safe option.

Reply:
But then we may find that the mode itself needs rethinking (what if Andrea puts the `InAppSearch` component that looks like the one on the interstitial page on some other, not interstitial, page?

For now, let's just rely on the mode to determine the height of the input. We can always change it if we come across surprises in the future. My suspicion is that in that case we will likely need to rethink the mode as well (or perhaps add another one).

Reply:
✅